src/axios.ts

```diff
@@ -193,6 +200,30 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
     return this.get(`/workspaces/${workspace}/tables/${table}/metadata`);
   };
 
+  Proto.uploadNetwork = async function(workspace: string, network: string, options: FileUploadOptionsSpec, config?: AxiosRequestConfig): Promise<AxiosResponse<Array<{}>>> {
+    const headers = config ? config.headers : undefined;
+    const params = config ? config.params : undefined;
+    const { type, data, url } = options;
+    const s3ffClient = new S3FileFieldClient({
+      baseUrl: `${url}/api/s3-upload/`,
+      // onProgress: onUploadProgress, // This argument is optional
+      apiConfig: this.defaults,
+    });
+
+    const fieldValue = await s3ffClient.uploadFile(
+      data as File,
+      'api.Upload.blob'
+    );
+
+    return this.post(`/workspaces/${workspace}/uploads/${type}/`, {
+      ...config,
+      headers: { ...headers, 'Content-Type': 'text/plain' },
+      ...params,
+      field_value: fieldValue.value,
+      network_name: network
+    });
+  };
```

Comment:
Similar to my other suggestions in `uploadTable`

```suggestion
  Proto.uploadNetwork = async function(workspace: string, network: string, options: FileUploadOptionsSpec): Promise<AxiosResponse<Array<{}>>> {
    const { type, data } = options;
    const s3ffClient = new S3FileFieldClient({
      baseUrl: `${this.baseURL}/api/s3-upload/`,
      apiConfig: this.defaults,
    });

    const fieldValue = await s3ffClient.uploadFile(data, 'api.Upload.blob');
    return this.post(`/workspaces/${workspace}/uploads/${type}/`, {
      field_value: fieldValue.value,
      network_name: network
    });
  };
```

Reply:
Addressed in a344b21, 846d1f3, 1bc301a, and ad03431.